README.md

```diff
@@ -9,21 +9,59 @@ server is deployed at:
 
 https://go-acd.appspot.com
 
-# Install
+# Usage
+
+In order to use this library, you must authenticate through the token server. A  (e.g. https://go-acd.appspot.com).
 
-This project is go-gettable:
+Get and build:
 
 ```
-go get gopkg.in/acd.v0/...
+$ go get gopkg.in/acd.v0/cmd/acd
+$ go build gopkg.in/acd.v0/cmd/acd
```

Comment:
Nope.

Also note that, in the output you gave above, you do a *get* but then magically already have a binary without building. Your image must already have a fully constituted GOPATH.